luigi/worker.py

```
@@ -238,8 +238,11 @@ class DequeQueue(collections.deque):
     deque wrapper implementing the Queue interface.
     """
 
-    put = collections.deque.append
-    get = collections.deque.pop
+    def put(self, *args, **kwargs):
```

Omit **kwargs? Better than including them to be silently dropped if passed in.